torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -1261,6 +1261,15 @@ const std::vector<std::string> functions = {
                 return grad_output * mask, None, None
             return torch.threshold(self, threshold, value), backward
 
+        def softplus(self,
+                      beta: number,
+                      threshold: number):
+            result = torch.softplus(self, beta, threshold)
+            def backward(grad_output):
+                z = torch.exp(result * beta)
```

Comment:
This actually replicates an imperfect implementation of softplus backward in order to match with eager-mode results.

upstream issue:
[https://github.com/pytorch/pytorch/issues/55587](https://github.com/pytorch/pytorch/issues/55587)